client/src/components/home/Home.jsx

```diff
@@ -1,14 +1,62 @@
-import React from "react";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import React,  { useState } from "react";
+import Login from "../login/Login";
+import SidePanel from "../navBar/SidePanel";
 import PersonalDashboard from "../personalDashboard/personalDashboard";
-import "../home/home.css";
+import AllModulesPage from "../All_Modules_Page/AllModulesPage";
+import Management from "../management/Management"
 import Footer from "../footer/Footer";
+import "../home/home.css";
+import { useAuth0 } from "@auth0/auth0-react";
+import ProfilePage from "../login/ProfilePage";
 
 const Home = () => {
+  const [refreshModalData, setRefreshModalData] = useState(false);
+  const { isAuthenticated } = useAuth0();
+
+
   return (
-    <div>
-      <PersonalDashboard />
-      <Footer />
-    </div>
+    <BrowserRouter>
+      {isAuthenticated ? (
+        <div className="flex flex-col">
+          <div className="flex flex-row">
+            <SidePanel />
+
+            <Routes>
+              <Route
+                path="/"
+                element={
+                  <PersonalDashboard
+                    refreshModalData={refreshModalData}
+                    setRefreshModalData={setRefreshModalData}
+                  />
+                }
+              />
+
+              <Route
+                path="/AllModulesPage"
+                element={
+                  <AllModulesPage
+                    refreshModalData={refreshModalData}
+                    setRefreshModalData={setRefreshModalData}
+                  />
+                }
+              />
+
+              <Route path="/management" element={<Management />} />
+              <Route path="/profilePage" element={<ProfilePage />} />
```

Comment:
Question: when/ how do users navigate here?

Reply:
there are (isAuthenticated) hooks from auth0 in line 20 checking as if isAuthenticated or  if not will local it to login only in line 56